composer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "jstoone/html-form-validator",
+    "description": "Validate a form based on it's html data attributes.",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Jakob Steinn",
+            "email": "jakobsteinn@protonmail.com"
+        }
+    ],
+    "require": {
+        "php": ">=5.5.9",
```

Comment:
@Ocramius, that sounds like a good idea. Although I need to get more into generators to see the exact benefit. Can you mention in which case/part of the library you see a good use-case for generators?